root/programs/VB/Frameworks/Infrastructure/Business/AsyncProcessingService/AsyncProcessingServiceParameterValue.vb

```diff
@@ -0,0 +1,192 @@
+﻿'**********************************************************************************
+'* Copyright (C) 2007,2014 Hitachi Solutions,Ltd.
+'**********************************************************************************
+
+#Region "Apache License"
+
+' Licensed under the Apache License, Version 2.0 (the "License");
+' you may not use this file except in compliance with the License. 
+' You may obtain a copy of the License at
+'
+' http://www.apache.org/licenses/LICENSE-2.0
+'
+' Unless required by applicable law or agreed to in writing, software
+' distributed under the License is distributed on an "AS IS" BASIS,
+' WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+' See the License for the specific language governing permissions and
+' limitations under the License.
+'
+#End Region
+
+'**********************************************************************************
+'* クラス名            :AsyncProcessingServiceParameterValue.cs
+'* クラス名クラス名     :
+'*
+'* 作成者              :Supragyan
+'* クラス日本語名       :
+'* 更新履歴
+'*  Date:        Author:          Comments:
+'*  ----------  ----------------  -------------------------------------------------
+'*  11/28/2014   Supragyan        Paramter Value class for Asynchronous Processing Service
+'*  04/15/2015   Sandeep          Changed datatype of ProgressRate to decimal.
+'**********************************************************************************
+' System
+Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+Imports System.Text
+
+'業務フレームワーク
+Imports Touryo.Infrastructure.Business.Common
+Imports Touryo.Infrastructure.Business.Util
+Imports System.Reflection
+
+Namespace AsyncProcessingService
+	''' <summary>
+	''' Paramter Value class for Asynchronous Processing Service
+	''' </summary>
+	Public Class AsyncProcessingServiceParameterValue
+		Inherits MyParameterValue
+		''' <summary>汎用エリア</summary>
+		Public Obj As Object
+
+		''' <summary>TaskId</summary>
+		Public TaskId As Integer
+
+		''' <summary>UserId</summary>
+		Public UserId As String
+
+		''' <summary>ProcessName</summary>
+		Public ProcessName As String
+
+		''' <summary>Data</summary>
```

Comment:
It can also apply to other items (TaskId, UserId, ProcessName, etc.).
I think that it should apply to description of　AsyncProcessingServiceParameterValue.
(However, it can be understood when read [specifications](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents/1_User_Guide/en/AsyncProcessingService) and procedures (in development).)
I suggest that you register this improvement to "issue" as backlog as necessary.